index.html

```diff
@@ -0,0 +1,87 @@
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Первый Code Jam</title>
+    <link rel="stylesheet" 
+    href="//cdn.rawgit.com/necolas/normalize.css/master/normalize.css">
+    <link rel="stylesheet" href="style.css">
+  </head>
+  <body>
+    <header>
+      <span class="header-sitename">Резюме</span>
+      <nav>
+        <ul>
+          <li class="nav-link"><a href="#about">О себе</a></li>
+          <li class="nav-link"><a href="#skills">Навыки</a></li>
+          <li class="nav-link"><a href="#code">Примеры</a></li>
+          <li class="nav-link"><a href="#exp">Опыт</a></li>
+          <li class="nav-link"><a href="#education">Образование</a></li>
+          <li class="nav-link"><a href="#english">English</a></li>
+        </ul>
+      </nav>
+    </header>
+    <main>
+      <article>
+        <h2 id="about">О себе</h2>
+        <img class="avatar" src="me.jpg" width="240" height="250" alt="my photo">
+        <p>
+          Меня зовут <strong>Коробейников Евгений</strong> и я хочу стать 
+          full-stack web-разработчиком.
+        </p>
+        <p>
+          Обладаю критическим складом ума, мне легко даётся изучение нового,
+          особенно в сфере IT. Умею анализировать проблемы и находить
+          оптимальные варианты их решения, исходя из поставленных задач.
+        </p>
+        <h2 id="skills">Навыки и умения</h2>
+        <p>
+          <ul>
+            <li>
+              C/C++
+            </li>
+            <li>
+              Arduino
+            </li>
+            <li>
+              HTML/CSS/JS Basics
+            </li>
+          </ul>
+        </p>
+        <h2 id="code">Примеры кода</h2>
+        <p>
+          Примеры кода можно найти в 
+          <a href="https://github.com/evgenijkorob/code-examples">репозитории</a>
+          или в других репозиториях
+          <a href="https://github.com/evgenijkorob/">моего github-аккаунта</a>.
+        </p>
+        <h2 id="exp">Опыт</h2>
+        <p>
+          Написал две игры на C/C++, вторую забросил. Также писал "змейку" на C
+          для микроконтроллера на базе платы Arduino. Писал на нём много
+          лабораторных и курсовых работ в университете.
+        </p>
+        <h2 id="education">Образование</h2>
+        <p>
+          Окончил <strong>МГОЛ №3</strong>, сейчас учусь в 
+          <strong>БГУИР</strong> на инженера-электроника-программиста.
+          Основы HTML/CSS/JS изучал на 
+          <a href="https://developer.mozilla.org/ru/">сайте</a>
+          MDN и
+          <a href="learn.javascript.ru">learn.javascript.ru</a>
+        </p>
+        <h2 id="english">English-level</h2>
+        <p>
+          Английский изучал с 1-го класса школы. Оценочный уровень языка
+          <strong>A2/B1</strong>.
+        </p>
+      </article>
+    </main>
+    <footer>
+      <h4 id="contacts">Мои контакты</h4>
+      <p>Моб. тел.(МТС): +375(29)29-38-541</p>
+      <p>E-mail: evgenij.k.wie@gmail.com</p>
+    </footer>
```

Comment:
```
      <p>Моб. тел.(МТС): +375(29)29-38-541</p>
      <p>E-mail: evgenij.k.wie@gmail.com</p>
```
You could wrap phone and email in tag `<a>` with specific href
```
<a href="tel:+3752938541">+375(29)29-38-541</a>
<a href="mailto:evgenij.k.wie@gmail.com>evgenij.k.wie@gmail.com1</a>
```
to add more interactivity